src/noise.c

```diff
@@ -69,6 +69,7 @@ int main_noise(int argc, char* argv[])
 
 	float stdev = sqrtf(var);
 
+#pragma omp parallel for
```

Comment:
Good point, removed.

Jon

On Wed, Jun 15, 2016 at 12:06 AM, Martin Uecker <notifications@github.com>
wrote:

> In src/noise.c
> <https://github.com/mrirecon/bart/pull/48#discussion_r67109883>:
>
> > @@ -69,6 +69,7 @@ int main_noise(int argc, char* argv[])
> >
> >  	float stdev = sqrtf(var);
> >
> > +#pragma omp parallel for
>
> This makes 'noise' indeterministic, which I think is bad and also defeats
> the purpose of rinit. We could make it an option (e.g. using #pragma omp
> for if (flag)) or think of another way to parallelize deterministically
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/mrirecon/bart/pull/48/files/91044d08cbd32bbd5c37d7edb0f0bc6c02aba64a..df166ac70e85babc8611854307695a628ad69137#r67109883>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AIUBa61or_F2UdXJ8x9ecbfGRzYJHWxjks5qL6RagaJpZM4I11-m>
> .
>